libdnf/hy-locale.c

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2017 Red Hat, Inc.
+ *
+ * Licensed under the GNU Lesser General Public License Version 2.1
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ */
+
+#include "hy-locale-private.h"
+
+void
+hy_locale_init(void)
```

Comment:
It's the application's responsibility to call this, not ours right?  I'm pretty sure it's a baseline requirement for GLib libraries to have the codeset be `UTF-8` anyways.  So why not just have `dnf-cmd.c` call `bind_textdomain_codeset(GETTEXT_PACKAGE, "UTF-8");` directly?

Reply:
@cgwalters I removed hy_locale_init().